src/PrestaShopBundle/Service/TranslationService.php

```diff
@@ -112,6 +116,8 @@ private function getResourcesDirectory()
      */
     public function getTranslationsCatalogue($lang, $type, $theme, $search = null)
     {
+        $translationCatalogueBuilder = $this->container->get('prestashop.translation.builder.translation_catalogue');
+
```

Comment:
```suggestion
```

Reply:
You mean "no need variable" ?

Reply:
The variable looks unused

Reply:
Yes, it's unused.

Reply:
Ok. I thought you was speaking about listDomainTranslation method. Fixed